help_function/src/onedpl_test_tagged_pointer.cpp

```diff
@@ -0,0 +1,153 @@
+// ====------ onedpl_test_tagged_pointer.cpp---------- -*- C++ -* ----===////
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//
+// ===----------------------------------------------------------------------===//
+
+// Adapted from onedpl_test_device_ptr.cpp
+
+#include <oneapi/dpl/algorithm>
+#include <oneapi/dpl/execution>
+#include <oneapi/dpl/iterator>
```

Comment:
is `#include <oneapi/dpl/iterator>` required?

Reply:
Removed